packages/browser-wallet/src/background/index.ts

```diff
@@ -109,6 +109,13 @@ chrome.storage.local.onChanged.addListener((changes) => {
     }
 });
 
+chrome.storage.session.onChanged.addListener((changes) => {
+    if (ChromeStorageKey.IsRecovering in changes) {
```

Comment:
Should it check the value, or should it call `startRecovery` even when `IsRecovering === false`.